pgjdbc/src/test/java/org/postgresql/test/jdbc3/Jdbc3CallableStatementTest.java

```diff
@@ -45,6 +45,10 @@ public void setUp() throws Exception {
         "CREATE OR REPLACE FUNCTION myiofunc(a INOUT int, b OUT int) AS 'BEGIN b := a; a := 1; END;' LANGUAGE plpgsql");
     stmt.execute(
         "CREATE OR REPLACE FUNCTION myif(a INOUT int, b IN int) AS 'BEGIN a := b; END;' LANGUAGE plpgsql");
+    stmt.execute(
+            "CREATE OR REPLACE FUNCTION mynoparams() returns int AS 'BEGIN return 0; END;' LANGUAGE plpgsql");
```

Comment:
Would you please use non-zero result as it might be confused with `NULL` when inspected via `getInt`?